clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -587,7 +587,7 @@ static void InitializeStandardPredefinedMacros(const TargetInfo &TI,
     if (LangOpts.getSYCLVersion() == LangOptions::SYCL_2017)
       Builder.defineMacro("CL_SYCL_LANGUAGE_VERSION", "121");
     else if (LangOpts.getSYCLVersion() == LangOptions::SYCL_2020)
-      Builder.defineMacro("SYCL_LANGUAGE_VERSION", "202001");
+      Builder.defineMacro("SYCL_LANGUAGE_VERSION", "202012");
```

Comment:
I don't know this code well, but it seems strange that the `L` suffix is missing.  Should it be like this instead?

```suggestion
      Builder.defineMacro("SYCL_LANGUAGE_VERSION", "202012L");
```

Reply:
Thanks.  Seems like this is resolved now.